### PR TITLE
Cleanup yaml in preparation for molecule testing

### PR DESCRIPTION
**Pull request by tyll**
This is a preparation for #107

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (pcahyna) [inline .travis.yml]: Do you need this now? It apparently causes a conflict. I think you can add it later, given that #107 contains
```yaml
  document-start: disable
```

Comment (tyll) [reply .travis.yml]: I will resolve the conflict. It was already part of #106

**Comment by tyll** [comment]
[citest bad]

**Comment by tyll** [comment]
[citest bad]

**Pull request merged**